src/Functional/Entries.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/**
+ * @package   Functional-php
+ * @author    Hugo Sales <hugo@hsal.es>
+ * @copyright 2021 Lars Strojny
+ * @license   https://opensource.org/licenses/MIT MIT
+ * @link      https://github.com/lstrojny/functional-php
+ */
+
+namespace Functional;
+
+use Functional\Exceptions\InvalidArgumentException;
+use Traversable;
+
+/**
+ * Inspired by JavaScript's `Object.entries`, and Python's `enumerate`,
```

Comment:
```suggestion
 * Inspired by JavaScript’s `Object.entries`, and Python’s `enumerate`,
```

Reply:
Is that U+2019: RIGHT SINGLE QUOTATION MARK? Why that change?

Reply:
It's typographically correct.